daml-lf/interpreter/src/main/scala/com/digitalasset/daml/lf/speedy/SBuiltin.scala

```diff
@@ -1008,7 +977,9 @@ object SBuiltin {
       // check if we find it locally
       machine.ptx.keys.get(gkey) match {
         case Some(mbCoid) =>
-          machine.ctrl = CtrlValue(SOptional(mbCoid.map(SContractId)))
+          machine.returnValue = SOptional(mbCoid.map { coid =>
+            SContractId(coid)
+          })
```

Comment:
@nickchapman-da @remyhaemmerle-da What's _operational_ the difference between
```scala
mbCoid.map(SContractId)
```
and
```scala
mbCoid.map { coid => SContractId(coid) }
```
In other words, why would you eta-expand? I've seen changes like this in other places too.

Reply:
I do not think there is any operational difference. 
However  `machine.returnValue = SOptional(mbCoid.map(SContractId))` is more idiomatic.